desktop-environment.el

```diff
@@ -182,6 +208,49 @@ portion of the screen."
   (message "New volume value: %s" (desktop-environment-volume-get)))
 
 
+;;; Helper functions - keyboard backlight
+(defun desktop-environment-keyboard-backlight-percent ()
+  "Return the new keyboard backlight value as a % instead
+of a single integer."
+  (if (eq (desktop-environment-keyboard-backlight-get) 0)
```

Comment:
what about calling `desktop-environment-keyboard-backlight-get` only once and storing its value in a `let` variable?

Reply:
Great idea. Done.